pkg/machines/libvirt.es6

```diff
@@ -237,6 +237,19 @@ LIBVIRT_PROVIDER = {
         };
     },
 
+    CHANGE_NETWORK_STATE ({ name, networkMac, state, connectionName }) {
+        logDebug(`${this.name}.CHANGE_NETWORK_STATE(${name}.${networkMac} ${state}):`);
+        return dispatch => {
+            spawnVirsh({connectionName,
+                method: 'CHANGE_NETWORK_STATE',
+                failHandler: buildFailHandler({ dispatch, name, connectionName, message: _("CHANGE NETWORK STATE action failed")}),
+                args: ['domif-setlink', name, networkMac, state]
+            }).then(() => {
+                dispatch(getVm(connectionName, name));
```

Comment:
Is this needed? Wouldn't be a change-event received?

Reply:
Change-event doesn't work, thus , yes, this is needed